cap_client/cap_api.py

```diff
@@ -135,24 +135,25 @@ def types(self):
 
     def get(self, pid=None, all=False):
         """Retrieve one or all analyses from a user."""
-        if pid or all:
-            url = urljoin('deposits/', pid)
-        else:
-            user_id = self.me().get('id', '')
-            url = urljoin('deposits/', '?q=_deposit.created_by:{}'.format(
-                user_id))
+        url = urljoin('deposits/', pid) if pid or all \
+            else urljoin('deposits/', '?q=&by_me=True')
```

Comment:
just one small thing, i know it was copied from previous code, but let's improve ;)
you don't need `urljoin` here, read what it does [here](https://docs.python.org/3/library/urllib.parse.html#urllib.parse.urljoin)

Reply:
Sometimes PID is `None`, which means that either we keep the first `urljoin` (that keeps the url `deposit/`, despite the pid), or we remove it, and make the default pid field empty `default=''` in the command definition (at `cli/cli.py`), for both get and get_shared. 

Which one do you prefer? Or a third way if you have in mind.

Reply:
first urljoin is ok, only second one doesnt make sense